ci/scripts/execute_redis_tests.sh

```diff
@@ -78,7 +78,10 @@ failCount=0
 --single unit/type/zset \
 --single unit/quit \
 --single unit/pubsub \
---single unit/dump
+--single unit/dump \
+--single unit/type/list \
+#--single unit/type/list-2 \ TODO enable when GEODE-10116 merged
+#--single unit/type/list-3 TODO enable when GEODE-10160 is fixed
```

Comment:
These comments are a little confusing, as this PR is for GEODE-10116, so it's strange to have a comment saying effectively "enable once this PR is merged", and GEODE-10160 is fixing bucket sizing, which seems like it shouldn't have an impact on unit-level tests.

Reply:
The one PR number is wrong, and I'll fix it. The list-2 and list-3 tests are stress tests. The first one needs LTRIM and the server blows up in list-3 because of sizing issues.

Reply:
In terms of a description - all tests have been enabled for the List commands supported as of now. The only blocking command we support, for example, is BLPOP so tests for that have been enabled, but not BRPOP, BRPOPLPUSH, etc. Tests that depend on unsupported command are all disabled, and a couple of tests that fail due to outstanding bugs are disabled with TODO comments.

Because LSET returns a slightly different error in one test than native Redis, the error condition has been modified.